### ISSUE TITLE
%opaque not working for WMF with text with transparent background

### ISSUE THREAD

**Issue by RonnyPixel** [ImageMagick/ImageMagick#7295]
### ImageMagick version

7.1.1-27 Q16-HDRI x64 0225ac4:20240121

### Operating system

Windows

### Operating system, version and so on

11

### Description

magick identify -format "%[opaque]" .\text.wmf
True
[text.zip](https://github.com/ImageMagick/ImageMagick/files/15229815/text.zip)

### Steps to Reproduce

magick identify -format "%[opaque]" .\text.wmf

### Images

![image](https://github.com/ImageMagick/ImageMagick/assets/169125729/88d6b488-0624-474b-b7d8-8fefd75bb7ee)

**Comment by snibgo** [comment]
It seems the problem isn't in %[opaque], but a mis-reading of the file test.wmf. I test with the simple command:
```
magick test.wmf out.png
```

When using IM v7.1.1-28 (pre-built binary) on Windows 11, the output is 32173x13547 pixels, entirely opaque white.

When using IM v7.1.1-27 (Beta) built by me, without libwmf, the output is 912x384 pixels, entirely opaque, mostly either white or black.

When viewing text.wmf with Windows Paint, the result is similar to IM v7.1.1-27, but black characters on a transparent background.

**Comment by snibgo** [comment]
The size problem (32173x13547 pixels) can be fixed by supplying a sensible density, eg:
```
magick -density 100 text.wmf t.png
```
The result is grayscale, black text on a white background. This suggests that the WMF contains a problematic density.

We are getting the defaults background, which is white. We should be able to change this with `-background None`, eg:
```
magick -density 100 -background None text.wmf t.png
```
However, the result is entirely opaque black.

We can try a different background:
```
magick -verbose -background sRGB(50%,75%,95%,0.5) -density 100 text.wmf t4.png
```
The result is black text on an opaque blueish background. The background *colour* is correct, but there is no transparency. I don't know why not. Possibly a bug in IM or libwmf or something.

**Comment by RonnyPixel** [comment]
Thank you for looking into this. Do you know if there's an easy way to tell
if a wmf file contains a bitmap or only vector graphics? (so I could assume
transparency from that) My only estimate now is to read the META header and
look at the size of the max record, number of objects and see if the dpi
value is inflated.

**Comment by snibgo** [comment]
I last used WMF in the 1990s, and can't remember any details. https://en.wikipedia.org/wiki/Windows_Metafile may be useful.

We can rasterize the WMF twice, at the same density but with different backgrounds. Then any pixels that are different must be background, so we can use that for opacity. Windows BAT syntax:
```
magick ^
  ( -density 100 -background Red text.wmf +write mpr:REDBK ) ^
  ( -density 100 -background Lime text.wmf ) ^
  -compose Difference -composite ^
  -negate ^
  mpr:REDBK ^
  +swap ^
  -alpha off -compose CopyOpacity -composite ^
  out.png
```

**Comment by RonnyPixel** [comment]
Yes it's some files from old powerpoint presentations. Thanks for the
suggestion!
